torchtitan/experiments/simple_fsdp/backend.py

```diff
@@ -8,11 +8,20 @@
 
 import torch
 
+from .job_config import Compile as CompileConfig
 
-def get_compile_backend(backend_name: str) -> Union[str, callable]:
+
+def get_compile_backend(
+    compile_config: CompileConfig, bucket_module_name: list[list[str] | str]
```

Comment:
maybe rename to `fsdp_buckets`?
not sure what will happen if it's in DDP / HSDP mode
```suggestion
    compile_config: CompileConfig, fsdp_buckets: list[list[str] | str]
```